mediathread/taxonomy/features/taxonomy.feature

```diff
@@ -216,41 +216,170 @@ Feature: Taxonomy
         I save the term
         I wait until the "Red" rename is complete
 
-        Then there is a "Blue" term
+        Then There is a "Blue" term
         Then there is no "Red" term
-        
+
         Finished using Selenium
-        
+
     Scenario: taxonomy.feature 9. Create from onomy
         Using selenium
         Given I am test_instructor in Sample Course
-        
+
+        # shortcut to taxonomy
+        When I access the url "/taxonomy/"
+        Given the taxonomy workspace is loaded
+        I see "Create Concept"
+
+        # Create a taxonomy
+        When I create a new concept
+        I name the concept "Colors"
+        I create the concept
+        Then there is a "Colors" link
+
+        When I click the Import button
+        And specify the onomy url
+        And confirm the onomy import
+
+        Then There is a "Black" term
+        Then There is a "Blue" term
+        Then There is a "Green" term
+        Then There is a "Pastels" term
+        Then There is a "Purple" term
+        Then There is a "Red" term
+
+        Then there is a "Pastels" link
+        When I click the "Pastels" link
+        Then There is a "Light Blue" term
+        Then There is a "Light Green" term
+        Then There is a "Pink" term
+
+        Finished using Selenium
+
+    Scenario: taxonomy.feature 10. onomy delete and refresh
+        Using selenium
+        Given I am test_instructor in Sample Course
+
         # shortcut to taxonomy
         When I access the url "/taxonomy/"
         Given the taxonomy workspace is loaded
         I see "Create Concept"
-        
+
         # Create a taxonomy
         When I create a new concept
         I name the concept "Colors"
         I create the concept
         Then there is a "Colors" link
-        
+
         When I click the Import button
         And specify the onomy url
         And confirm the onomy import
 
-        Then there is a "Black" term
-        Then there is a "Blue" term
-        Then there is a "Green" term
-        Then there is a "Pastels" term
-        Then there is a "Purple" term
-        Then there is a "Red" term
-        
+        Then There is a "Black" term
```

Comment:
the step is @step(u'There is a "([^"]*)" term') So I capitalized it to correspond.